locales/fr.yml

```
@@ -24,11 +24,26 @@ fr:
       main-title: Nous sommes développeurs <span>Ruby/Rails</span>
       subtitle: experts en backend, MVPs &amp; open-source
     our-projects:
+      accorlocal:
+        description: Réservation de services de proximité dans les hôtels du groupe Accor. Développement back-end et back-office.
+        link: https://www.accorlocal.com/
+        stack: Rails / API Restfull / Rspec / PostgreSQL
```

`RESTful`, not `Restfull`... and `RSpec`, not `Rspec`... but I think we should even change it to:

```suggestion
        stack: Rails / API (Grape) / ActiveAdmin / RSpec / PostgreSQL
```